packages/ra-core/src/i18n/TranslationMessages.ts

```diff
@@ -119,6 +119,7 @@ export interface TranslationMessages extends StringMap {
             page_range_info: string;
             partial_page_range_info: string;
             page_rows_per_page: string;
+            items_per_page_lable: string;
```

Comment:
Since the translation key is provided by the user, and not by the `<Pagination>` component, it can be whatever key, so it doesn't belong in this file, but should rather be added in the [translations files](https://github.com/marmelab/react-admin/blob/master/examples/simple/src/i18n/en.ts#L3) of the demo.